roles/custom/matrix-user-verification-service/templates/.env.j2

```diff
@@ -0,0 +1,14 @@
+UVS_ACCESS_TOKEN={{ matrix_user_verification_service_uvs_access_token }}
+UVS_HOMESERVER_URL={{ matrix_user_verification_service_uvs_homeserver_url }}
+UVS_DISABLE_IP_BLACKLIST={{ matrix_user_verification_service_uvs_disable_ip_blacklist }}
+
+{% if matrix_user_verification_service_uvs_auth_token is defined and matrix_user_verification_service_uvs_auth_token|length %}
+  UVS_AUTH_TOKEN={{ matrix_user_verification_service_uvs_auth_token }}
+{% endif %}
+{% if matrix_user_verification_service_uvs_openid_verify_server_name is defined and matrix_user_verification_service_uvs_openid_verify_server_name|length %}
+  UVS_OPENID_VERIFY_SERVER_NAME={{ matrix_user_verification_service_uvs_openid_verify_server_name }}
+{% endif %}
+{% if matrix_user_verification_service_uvs_log_level is defined and matrix_user_verification_service_uvs_log_level|length %}
+  UVS_LOG_LEVEL={{ matrix_user_verification_service_uvs_log_level }}
+{% endif %}
```

Comment:
Why `is defined`? Let's ensure all variables are defined in the role itself and have sane values.

`|length` checks are probably also unnecessary in at least a few of these (e.g. `matrix_user_verification_service_uvs_log_level`). Let's set a sane default value in the variable and just use it here without any `if`s at all.

Maybe this is also applicable to `matrix_user_verification_service_uvs_auth_token`.

I understand that for `matrix_user_verification_service_uvs_openid_verify_server_name` we may need some if (discussed in some other PR comments).